modular_zubbers/code/modules/clothing/suits/jacket.dm

```diff
@@ -63,3 +63,130 @@
 	worn_icon = 'modular_zubbers/icons/donator/skyymed_worn.dmi'
 	armor_type = /datum/armor/toggle_labcoat
 	toggle_noun = "zipper"
+
+/obj/item/clothing/suit/jacket/tailcoat //parent type
+	name = "tailcoat"
+	desc = "A coat usually worn by bunny themed waiters and the like."
+	worn_icon = 'modular_zubbers/icons/mob/clothing/suits/jacket.dmi'
+	icon = 'modular_zubbers/icons/obj/clothing/suits/jacket.dmi'
+	icon_state = "tailcoat"
+	greyscale_colors = "#39393f"
+	greyscale_config = /datum/greyscale_config/tailcoat
+	greyscale_config_worn = /datum/greyscale_config/tailcoat_worn
+	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/suit/jacket/tailcoat/bartender
+	name = "bartender's tailcoat"
+	desc = "A coat usually worn by bunny themed bartenders. It has an interior holster for firearms and some extra padding for minor protection."
+	icon_state = "tailcoat_bar"
+	greyscale_colors = "#39393f#ffffff"
+	greyscale_config = /datum/greyscale_config/tailcoat_bar
+	greyscale_config_worn = /datum/greyscale_config/tailcoat_bar_worn
+	armor_type = /datum/armor/tailcoat_bar
+
+/datum/armor/tailcoat_bar
+	melee = 15
+	bullet = 10
+	laser = 10
+	energy = 15
+	fire = 20
+	bomb = 10
+	acid = 20
+	wound = 5
+
+/obj/item/clothing/suit/jacket/tailcoat/bartender/Initialize(mapload) //so bartenders can use cram their shotgun inside
+	. = ..()
+	allowed += list(
+		/obj/item/ammo_box,
+		/obj/item/ammo_casing,
+		/obj/item/gun/ballistic,
+		/obj/item/gun/energy,
+	)
+
+/obj/item/clothing/suit/jacket/tailcoat/syndicate
+	name = "suspicious tailcoat"
+	desc = "A oddly intimidating coat usually worn by bunny themed assassins. It's reinforced with some extremely flexible lightweight alloy. How much did they pay for this?"
+	icon_state = "tailcoat_syndi"
+	armor_type = /datum/armor/tailcoat_syndi
+	greyscale_config = null
+	greyscale_config_worn = null
+	greyscale_colors = null
+
+
+/datum/armor/tailcoat_syndi
+	melee = 30
+	bullet = 25
+	laser = 30
+	energy = 35
+	fire = 50
+	bomb = 20
+	acid = 50
+	wound = 10
```

Comment:
Isn't there already a syndie armor datum with similar values in the code? Thinking like `/datum/armor/wintercoat_syndicate` or `/datum/armor/navywarden_syndicate` could be used instead of creating another datum, both have fairly similar values too